src/pkg/cli/client/byoc/aws/byoc.go

```
@@ -747,7 +747,7 @@ func (b *ByocAws) Follow(ctx context.Context, req *defangv1.TailRequest) (client
 			if err := ecs.WaitForTask(ctx, taskArn, 2*time.Second); err != nil {
 				if stopWhenCDTaskDone || errors.As(err, &ecs.TaskFailure{}) {
 					time.Sleep(2 * time.Second) // make sure we got all the logs from the task/ecs before cancelling
-					cancel(pkg.ErrDeploymentFailed{Service: "Defang CD", Message: err.Error()})
+					cancel(pkg.ErrDeploymentFailed{Service: "defang-cd", Message: err.Error()})
```

fabric debug will fail early if the service name is empty or does not meet the naming requirements (lowercase a-z, 0-9, and hyphen)

---

Let's remove the `Service: "defang-cd"`. I have bad experiences with making pseudo service names like that.